stl/inc/xstring

```diff
@@ -3616,6 +3616,25 @@ public:
         return _Mypair._Myval2._Myres;
     }
 
+#if _HAS_CXX20
+    void reserve(_CRT_GUARDOVERFLOW const size_type _Newcap) { // determine new minimum length of allocated storage
+        if (_Mypair._Myval2._Myres < _Newcap) { // reallocate to grow

```

Comment:
We conventionally prefer to early-return and reduce indentation, which is feasible now that this function has a single conditional.

Reply:
Thanks! One question though, after the `if` block with the early return, do you normally have another return statement at the end of the function?
e.g.
```
void reserve(_CRT_GUARDOVERFLOW const size_type _Newcap) {
    if (_Mypair._Myval2._Myres >= _Newcap) {
        return;
    }
    // reallocate
    return; // <- is this return normally here?
}
```

Reply:
No, our convention is to flow off the end of `void`-returning functions.